src/index.ts

```diff
@@ -5202,16 +5202,30 @@ const shardusSetup = (): void => {
 
         const isStakeRelatedTx: boolean = isStakingEVMTx(transaction)
 
-        const isEIP2930 =
-          transaction instanceof AccessListEIP2930Transaction && transaction.AccessListJSON != null
         let isSimpleTransfer = false
-
         let remoteShardusAccount
         let remoteTargetAccount
         appData.requestNewTimestamp = true // force all evm txs to generate a new timestamp
 
-        if (isEIP2930 && isStakingEVMTx(transaction) === false) {
```

Comment:
we removed `isStakingEVMTx(transaction) === false` check altogether.. is that expected?